### ISSUE TITLE
Error while removing application credential in bin/delete_cluster.sh

### ISSUE THREAD

**Issue by berendt** [SovereignCloudStack/k8s-cluster-api-provider#243]
```
Removing application credential capi-testcluster-appcred ...
./bin/delete_cluster.sh: line 74: /home/ubuntu/tmp/clouds-no-capi-testcluster.yaml: No such file or directory
```

**Comment by garloff** [comment]
Same reason as #240: Missing path causes print-cloud.py not to be found.

**Comment by garloff** [comment]
Fixed by #247.
